rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/LocalInfrastructure.java

```diff
@@ -31,24 +31,19 @@
     @Configurable(description = "Maximum number of nodes to\nbe deployed on Resource Manager machine")
     private int maxNodes = DEFAULT_NODE_NUMBER;
     // number of nodes which can still be acquired
-    private AtomicInteger atomicMaxNodes;
+    private AtomicInteger acquiredNodes;
+    private AtomicInteger lostNodes;
+    private AtomicInteger handledNodes;
+    private AtomicBoolean commandLineStarted;
     @Configurable(description = "in ms. After this timeout expired\nthe node is considered to be lost")
     private int nodeTimeout = DEFAULT_TIMEOUT;
     @Configurable(description = "Additional ProActive properties")
     private String paProperties = "";
 
     /** To link a nodeName with its process */
-    private Hashtable<String, ProcessExecutor> nodeNameToProcess;
-    /**
-     * To link a deploying node url with a boolean, used to notify deploying
-     * node removal
-     */
-    private Hashtable<String, Boolean> isDeployingNodeLost;
-    /**
-     * Notifies the acquisition loop that the node has been acquired and that it
-     * can exit the loop
-     */
-    private Hashtable<String, Boolean> isNodeAcquired;
+    //private Hashtable<String, ProcessExecutor> nodeNameToProcess;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)